### DISCUSSION TITLE
Mojo Version, Porting localGPT to Mojo. 🔥

### DISCUSSION THREAD

**Discussion by gabrielkryss** [PromtEngineer/localGPT#553] [Ideas]
I don't know if its my place to say this, I personally feel, based of my skill level. But is there any talks about doing a Mojo port of this repository?

I am not sure about the current state of this repository, has people moved to another version of this project, a preferred forked version or whatever. If so maybe we can move the conversation there? If not I guess ignore this paragraph I guess.

**Comment by PromtEngineer** [comment]
@gabrielkryss thanks for bringing the idea. I think that will be great if we can port it to Mojo. 

I personally haven't looked at Mojo too much but if you or someone else in the community can help, I would love to do that.
